numba_cuda/numba/cuda/__init__.py

```diff
@@ -44,6 +46,16 @@
             "bindings."
         )
 
+if config.CUDA_ENABLE_PYNVJITLINK:
+    if USE_NV_BINDING:
+        warnings.warn(
+            "Explicitly enabling pynvjitlink is no longer necessary. "
+            "NVIDIA bindings are enabled. cuda.core will be used "
+            "in place of pynvjitlink."
+        )
+    else:
+        raise RuntimeError("nvJitLink requires the NVIDIA CUDA bindings. ")
```

Comment:
Because `config.CUDA_ENABLE_PYNVJITLINK` is enabled automatically if it's found in the environment, disabling the NVIDIA bindings now leads to this exception being hit.